backend/.env.example

```
@@ -308,6 +308,41 @@ TELEGRAM_TYPING_REFRESH_SECONDS=2
+# ============================================================================
+# CHANNELS: GOOGLE CHAT (single-user dogfood)
+# ============================================================================
+# Receives Google Chat events over a Pub/Sub *pull* subscription (no public
+# webhook needed) and replies via the Chat REST API. Leaving the service
+# account / project / subscription empty disables the channel — the pull
+# loop never starts. See app/channels/google_chat/.
+
+# Path to the Google Cloud service-account JSON (scopes: chat.bot + pubsub).
+GOOGLE_CHAT_SERVICE_ACCOUNT_FILE=
+# Google Cloud project that owns the Pub/Sub topic + subscription.
+GOOGLE_CHAT_PROJECT_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GOOGLE_CHAT_PROJECT_ID key should go before the GOOGLE_CHAT_SERVICE_ACCOUNT_FILE key

(UnorderedKey)